website/docs/r/spacecraft.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+subcategory: "Orbital"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spacecraft"
+description: |-
+  Manages a Spacecraft resource.
+---
+
+# azurerm_spacecraft
+
+Manages a Spacecraft.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "rg-example"
+  location = "West Europe"
+}
+
+resource "azurerm_spacecraft" "example" {
+  name                = "example-spacecraft"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = "westeurope"
+  norad_id            = "12345"
+  links {
+    bandwidth_mhz        = 100
+    center_frequency_mhz = 101
+    direction            = "Uplink"
+    polarization         = "LHCP"
+    name                 = "examplename"
+  }
+  tle_line_1 = "1 23455U 94089A   97320.90946019  .00000140  00000-0  10191-3 0  2621"
+  tle_line_2 = "2 23455  99.0090 272.6745 0008546 223.1686 136.8816 14.11711747148495"
```

Comment:
should we just make this a ordered list?
```suggestion
  tle[0] = "1 23455U 94089A   97320.90946019  .00000140  00000-0  10191-3 0  2621"
  tle[1] = "2 23455  99.0090 272.6745 0008546 223.1686 136.8816 14.11711747148495"
```

Reply:
These 2 fields are parameters of a spacecraft's orbit, and there will be only 2 lines, the combination of them can point to a certain spacecraft.  Making them a list could confuse the user a little bit.

Reply:
I still don't think theres a a reason to not make this a type list? as it would be more versatile and allow users to n pass-around an array vs 2 separate strings? min items x, max items 2

i don't think it would be confusing, espically maybe if we rename it to
```suggestion
  two_line_elements[0] = "1 23455U 94089A   97320.90946019  .00000140  00000-0  10191-3 0  2621"
  two_line_elements[0] = "2 23455  99.0090 272.6745 0008546 223.1686 136.8816 14.11711747148495"
```